Revamped Nukie Planet/Outpost!

choccy4 opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Basically! This PR completely replaces the current nukeops outpost with a new more flashy and modern one. Less like a cool hangout spot and more _evil_ terrorist lair...
## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
I thought that the present nukeops base is a bit bland, a bit _lame_ even. So, I decided to make a new one! Oh, and also it fixes some gripes about nukies e.g. not having a large amount of welding fuel - and, it has some fun stuff, like a place for a trader admeme, a prison, better dorms, a bigger bar, better firing range, more bonus gear (mainly power cells for mechs and shields) and an actual medbay!
## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
![image](https://github.com/user-attachments/assets/c100502b-e789-46c0-b265-2eb952245cca)
![image](https://github.com/user-attachments/assets/ffbdb593-45d5-4f62-a8d2-9f11975aa231)
![image](https://github.com/user-attachments/assets/00353959-2b04-477d-a3c0-0603913d621a)
![image](https://github.com/user-attachments/assets/00c17e1f-c22d-4bd8-a898-5afeec135bb0)
![image](https://github.com/user-attachments/assets/1d49995c-eb53-4490-909b-eb8586f8e772)
![image](https://github.com/user-attachments/assets/503259ac-30c7-45f8-80b0-2e09709ec705)
![image](https://github.com/user-attachments/assets/29db9e62-bd7a-4a0a-8d59-96008cd04503)

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->
:cl: choccy
- tweak: New starlight specific nukeops outpost!
- tweak: More gear on nukie base, mainly stuff that's good QoL!

choccy4 committed:
completely redesigns the nukie outpost

choccy4 committed:
alters the gamerule for nukeops

to make sure it points to the correct path for the new map

choccy4 committed:
changed the antique cells for microreactors

PubliclyExecutedPig commented:
I'd say in CL you don't need to add all the technical details, just the fact there is a new outpost. I'll start review in an hour or less

PubliclyExecutedPig commented:
I completely forgot to review it, gimme ~2ish hours

CringeCursed commented:
Sheeesh looks pretty cool 🔥🔥🔥

PubliclyExecutedPig commented:
idk if this is intentional but theres atmos on here despite space backround, i dont mind too much but just incase its unintended
![image](https://github.com/user-attachments/assets/ef3728e1-83b0-4c27-b621-ae5c0c18b695)

PubliclyExecutedPig commented:
![image](https://github.com/user-attachments/assets/6448fc38-142b-4866-9dbd-4f91c2f9d678)
Can't for the life of me figure out how to open these doors, just used the super door remote

PubliclyExecutedPig commented:
![image](https://github.com/user-attachments/assets/94dc505e-16d9-4cfa-87ca-0eef548a3d6a)
Remove this taser

PubliclyExecutedPig commented:
![image](https://github.com/user-attachments/assets/bcbc64a2-243b-4da2-9b88-442f00a37691)
remove this syringe gun

PubliclyExecutedPig commented:
after a full look around the map, if you remove all of the above i'll approve it.
also roundstart vestine and a plasma canister is scary but we'll see how it plays out

choccy4 commented:
to address all this! the background is intended, i changed it to non planet side because it's cool and also logic wise it's in a near orbit roid belt looking outward, the button for the doors is inside the back room - it's meant for admemes - the other non syndie things i'll remove and the round start vestine is like theyre terrorists of course they'll have illegal chemicals and also the can of plasma is intentionally locked so that nukies either have to access break it/steal an id 🐈

choccy4 commented:
also yo getting referenced by a random other community is funny

PubliclyExecutedPig commented:
> to address all this! the background is intended, i changed it to non planet side because it's cool and also logic wise it's in a near orbit roid belt looking outward, the button for the doors is inside the back room - it's meant for admemes - the other non syndie things i'll remove and the round start vestine is like theyre terrorists of course they'll have illegal chemicals and also the can of plasma is intentionally locked so that nukies either have to access break it/steal an id 🐈

once you make the commit to change the stuff ill approve the pr, i only mentioned vestine bc its normally an uplink thing

choccy4 committed:
removed taser, removed syringe gun

choccy4 committed:
fixes previous issues

changes syringe gear to clown gear, adds intellicard, adds interdyne defib, adds WEH! clothing

choccy4 committed:
fixed file formatting

PubliclyExecutedPig reviewed (approved):
ok

choccy4 commented:
> ok

gurted
🔥

choccy4 commented:
> > to address all this! the background is intended, i changed it to non planet side because it's cool and also logic wise it's in a near orbit roid belt looking outward, the button for the doors is inside the back room - it's meant for admemes - the other non syndie things i'll remove and the round start vestine is like theyre terrorists of course they'll have illegal chemicals and also the can of plasma is intentionally locked so that nukies either have to access break it/steal an id 🐈
> 
> once you make the commit to change the stuff ill approve the pr, i only mentioned vestine bc its normally an uplink thing

yeah, vestine guaranteed is odd but i thnik that its good since it makes the agent mroe fun and no one ever buys the kit anyway since its generally not worth it at all

choccy4 committed:
reverted claymore

choccy4 committed:
Merge branch 'ss14Starlight:Starlight' into nukie

Happyrobot33 commented:
choccy just double checking, this is clean for a merge?

choccy4 commented:
> choccy just double checking, this is clean for a merge?

let me check and i'll tell you soon, i need to grab the latest build and test it, but in theory - it should be good to go since it's only a map file

choccy4 commented:
> choccy just double checking, this is clean for a merge?

tested, seems to work perfectly fine atm, so i think its good to go!
*had to update to newest, glad you mentioned

choccy4 committed:
Merge branch 'ss14Starlight:Starlight' into nukie
